### DISCUSSION TITLE
did you have an idea of the next release?

### DISCUSSION THREAD

**Discussion by guyomog78** [ltb-project/self-service-password#467] [Q&A] [answered]
Hello,

I follow this projet sinc almost 1 year. I see that you do regulary update and since july, you compltely have changed the structure.
With this structure, it's no mmode possible to git clone and use the project as is.
I'm interresting to have recent version. I'm able to test a wile and maybe help for some stuff.

You have my best support for all you work.

Regards,

Guillaume

**Comment by coudot** [comment, marked as answer]
Hello,

we made a big change by using Smarty framework, you just need to install PHP Smarty to use the project from the master.

We indeed plan to release 1.4 soon.

**Reply by wmunguiam** [reply]
Hi Coudot,

It is a wonderfull project! In my case, Do I have some news about 1.4?

**Reply by coudot** [reply]
Release is in progress, hope this week.

**Comment by guyomog78** [comment]
Super,

Merci pour ton retour, je vais tester ça! si je vois des trucs, je
n'hésiterais pas à faire un retour ! ou à proposer une solution ;-)
